package nail_sensor_msgs.msgs;


message NailCollision
{

    required bool collision = 1;
  
}
